src/components/common/AppHeader.vue

```diff
@@ -3,11 +3,35 @@
+            />
+            <path
+              d="M172.268 76.238V67.4155H147.158V27.8495H137.656V76.238H172.268Z"
+            />
+            <path
+              d="M205.772 76.238H213.713V37.2151H204.211V57.0998C204.211 64.2257 201.022 69.0442 194.846 69.0442C188.331 69.0442 186.159 64.8365 186.159 58.3893V37.2151H176.726V60.8324C176.726 70.8766 181.612 77.1203 191.453 77.1203C198.239 77.1203 202.786 73.7949 204.89 68.5692L205.772 76.238Z"
+            />
+            <path
+              d="M231.218 56.489C231.218 48.888 234.476 44.2053 241.534 44.2053C248.321 44.2053 250.56 48.413 250.56 55.0638V76.238H260.061V52.7564C260.061 42.6443 254.632 36.1971 244.656 36.1971C237.598 36.1971 232.847 39.7261 230.608 45.1554L229.725 37.2151H221.717V76.238H231.218V56.489Z"
+            />
+            <path
+              d="M271.775 32.668C275.914 32.668 278.018 30.9035 278.018 26.5601C278.018 22.0809 275.847 20.52 271.775 20.52C267.771 20.52 265.599 22.0809 265.599 26.5601C265.599 31.1071 267.295 32.668 271.775 32.668ZM276.457 76.238V37.2151H266.888V76.238H276.457Z"
+            />
+            <path
+              d="M303.231 43.5945C309.882 43.5945 313.411 46.9878 314.158 52.96H291.355C292.372 46.9878 296.512 43.5945 303.231 43.5945ZM322.98 64.7008H312.8C311.307 67.8905 308.321 69.655 303.163 69.655C296.037 69.655 292.101 66.3296 291.287 60.2216H323.523V56.0818C323.523 45.4947 317.076 36.1971 303.095 36.1971C289.929 36.1971 281.989 44.5446 281.989 56.8283V57.5749C281.989 69.5872 288.504 77.3239 302.688 77.3239C313.411 77.3239 320.265 72.9126 322.98 64.7008Z"
+            />
+          </svg>
```

Comment:
I'm just curious, why do we switch to SVG for our logo? To not need to import it right?

Reply:
good question! it's to be able to set the color with a variable.